docs/kyma/05-01-overview.md

```diff
@@ -3,53 +3,69 @@ title: Overview
 type: Configuration
 ---
 
-## Default settings
+## Introduction
+
+Users can configure Kyma installation in two ways:
+  - Customize the list of the components to install
+  - Provide overrides that change configuration values used by one or more components
+
+The list of components to install is defined in the **Installation** Custom Resource.
+Overrides are defined as ConfigMap or Secret objects defined by the user before the installation starts.
+The Installer is the Kyma component that is responsible for reading and applying the configuration.
 
-During installation, the Installer applies onto a cluster or Minikube all components defined in the `installer-cr-cluster.yaml.tpl` or `installer-cr.yaml.tpl` templates with their configuration defined in `values.yaml` files. It also imports the configuration overrides defined in the `installer-config-cluster.yaml.tpl` and `installer-config-local.yaml.tpl` templates located under the `installation/resources` subfolder.
 
-> **NOTE:** The installation and configuration templates serve as the basis for creating the installation (`kyma-installer-local.yaml` and `kyma-installer-cluster.yaml`) and configuration (`kyma-config-local.yaml` and `kyma-config-cluster.yaml`) release artifacts.
+## Default settings
```

Comment:
Consider splitting cluster and local sections with a toggle switch.

Reply:
Done.